### ISSUE TITLE
ChunkingSink does not write Requests/Responses without body

### ISSUE THREAD

**Issue by schaluka** [zalando/logbook#842]
## Detailed Description

ChunkingSink should write requests/responses without a body. It should use the delegate sink directly as it only chunks the bodies.

## Your Environment

    * Version used: 2.1.0

## Reproducing
```java
// create a sink which prints to console
var sink = new ChunkingSink(new DefaultSink(new DefaultHttpLogFormatter(), new HttpLogWriter() {

	@Override
	public void write(Precorrelation precorrelation, String request) throws IOException {
		System.out.println(request);

	}


	@Override
	public void write(Correlation correlation, String response) throws IOException {
		System.out.println(response);
	}

}), 100);
// just use whatever does not matter for this test
var corr = new Correlation() {

	@Override
	public String getId() {
		return UUID.randomUUID().toString();
	}


	@Override
	public Instant getStart() {
		return Instant.now();
	}


	@Override
	public Instant getEnd() {
		return Instant.now();
	}


	@Override
	public Duration getDuration() {
		return Duration.ofSeconds(5);
	}

};
// will be printed
sink.write(corr, MockHttpRequest.create().withBodyAsString("bodyAsString"));
sink.write(corr, MockHttpRequest.create().withBodyAsString("bodyAsString"), MockHttpResponse.create().withBodyAsString("bodyAsString"));
// will not be printed
sink.write(corr, MockHttpRequest.create());
sink.write(corr, MockHttpRequest.create(), MockHttpResponse.create());
```

**Comment by whiskeysierra** [comment]
Thanks for the test case! I'll try to take a look soon.

**Comment by whiskeysierra** [comment]
Fixed in 2.2.1
